src/stories/components/organisms/Calendar.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Calendar } from '@/components';
+// import { UserType, CommentType } from '@/Models';
+import styled from '@emotion/styled';
+import dayjs from 'dayjs';
+
+export default {
+  title: 'Components/Organisms/Calendar',
+  component: Calendar,
+};
+
+const markedDates = [
+  dayjs('2022-01-01'),
+  dayjs('2022-01-04'),
+  dayjs('2022-01-05'),
+  dayjs('2022-01-12'),
+  dayjs('2022-01-15'),
+];
```

Comment:
이런 형식으로 넣는거군요ㅎㅎ 마킹되는 동그라미의 색깔은 하나로 통일하는게 좋을까요?! 
깃허브 잔디 같은 형식 다시 생각해봤는데 동그라미 하나로 표시되는거면 그냥 동일하게 가는 것도 좋을 것 같네요ㅎㅎ

Reply:
> 달력 패키지 찾다가 결국 달력을 만들어버리는 윤호님 클래스에 감탄하고 갑니다ㅎㅎ 한번 쫙 설명 부탁드려요 윤호님 🤩 너무 너무 고생 많으셨습니다 👏 디자인 전체적으로 깔끔하니 예쁜 것 같아요! 별건 아니지만 혹시 아래 사진에 월 선택 시 왼쪽 맨위에 `1월 2022` 이 부분 `2022년 1월` 이런식으로 위치 변경이 가능할까요?! 위에 제목이랑 통일되면 좋을 것 같아서요ㅎㅎ 피커 기능에 없거나 변경 불가능하다면 딱히 상관은 없을 것 같아요~~~ <img alt="달력" width="304" src="https://user-images.githubusercontent.com/77623643/150716028-cfbd57d8-8301-4680-b9f1-61b06def6e2a.png">

@dasssseul 

오 이부분은 미쳐 발견을 못했는데 다슬님 역시 디테일👍
어떻게 바꿔야할지 검색해서 찾아보았는데요 스택오버플로우에서 아래와 같은 사실을 알게되었습니다😂

> There is no normal way to do it. I've checked source code. I guess you can use some dirty hack like this below for example. It will not working forever:

일반적으로는 못바꾸는것 같은데 계속 인지하다가 바꿀수 있는 방법을 알게되면 다시 시도해볼겠습니다!

참고: https://stackoverflow.com/questions/68665803/how-to-customize-the-calendar-header-text-format-material-ui-v5-datepicker

Reply:
> 이런 형식으로 넣는거군요ㅎㅎ 마킹되는 동그라미의 색깔은 하나로 통일하는게 좋을까요?! 깃허브 잔디 같은 형식 다시 생각해봤는데 동그라미 하나로 표시되는거면 그냥 동일하게 가는 것도 좋을 것 같네요ㅎㅎ

@dasssseul 
저도 잔디 버전도 생각을 해봤는데요!! 2가지 고려사항이 있었습니다!
- 잔디의 컬러 레벨에 대한 정의 i.e) 컬러 레벨이 3개라면, 수행루틴을 몇개를 했을때 레벨을 정할지
- api 응답의 구조, api 응답 데이터 결과에 따라 데이터를 가공이 필요, 허나 지금은 api 응답 데이터의 정확한 형태를 모름🥲

저도 잔디버전이 더 직관적인 UI라고 생각해서 바꾸고 싶은데 우선은 최소 기능만을 생각해서 간단하게 구현하였습니다.
api 명세가 나오고 회의때 잔디 컬러 레벨에 대해 같이 의논해본 후 다음 리팩토링 대상에 추가해놓겠습니다!

Reply:
> 

네네 백둥이분들이랑도 한번 의논해보고 차차 정해봅시다 🙌